.github/CODEOWNERS

```diff
@@ -376,6 +376,8 @@ packages/react-components/react-color-picker-preview/library @microsoft/cxe-prg
 packages/react-components/react-color-picker-preview/stories @microsoft/cxe-prg
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33107&commitId=91bdf412ca89d7a33312d1001f3bbe3cd4627589&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|3| Changed|

</details>


  <details><summary><b>Charts-DonutChart</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Charts-DonutChart.Basic Dark Mode.chromium.png|0| Added|
|Charts-DonutChart.Basic RTL.chromium.png|0| Added|
|Charts-DonutChart.Basic.chromium.png|0| Added|
|Charts-DonutChart.Dynamic Dark Mode.chromium.png|0| Added|
|Charts-DonutChart.Dynamic RTL.chromium.png|0| Added|
|Charts-DonutChart.Dynamic.chromium.png|0| Added|

</details>


  <details><summary><b>Charts-HorizontalBarChart</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Charts-HorizontalBarChart.Basic Dark Mode.chromium.png|0| Added|
|Charts-HorizontalBarChart.Basic RTL.chromium.png|0| Added|
|Charts-HorizontalBarChart.Basic.chromium.png|0| Added|
|Charts-HorizontalBarChart.Variant Dark Mode.chromium.png|0| Added|
|Charts-HorizontalBarChart.Variant RTL.chromium.png|0| Added|
|Charts-HorizontalBarChart.Variant.chromium.png|0| Added|
|Charts-HorizontalBarChart.With Benchmark Dark Mode.chromium.png|0| Added|
|Charts-HorizontalBarChart.With Benchmark RTL.chromium.png|0| Added|
|Charts-HorizontalBarChart.With_Benchmark.chromium.png|0| Added|

</details>


  <details><summary><b>Charts-Legend</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Charts-Legend.Basic Dark Mode.chromium.png|0| Added|
|Charts-Legend.Basic RTL.chromium.png|0| Added|
|Charts-Legend.Basic.chromium.png|0| Added|
|Charts-Legend.Overflow Dark Mode.chromium.png|0| Added|
|Charts-Legend.Overflow RTL.chromium.png|0| Added|
|Charts-Legend.Overflow.chromium.png|0| Added|

</details>


  <details><summary><b>Charts-LineChart</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Charts-LineChart.Basic Dark Mode.chromium.png|0| Added|
|Charts-LineChart.Basic RTL.chromium.png|0| Added|
|Charts-LineChart.Basic.chromium.png|0| Added|
|Charts-LineChart.Events Dark Mode.chromium.png|0| Added|
|Charts-LineChart.Events RTL.chromium.png|0| Added|
|Charts-LineChart.Events.chromium.png|0| Added|
|Charts-LineChart.Gaps Dark Mode.chromium.png|0| Added|
|Charts-LineChart.Gaps RTL.chromium.png|0| Added|
|Charts-LineChart.Gaps.chromium.png|0| Added|
|Charts-LineChart.Multiple Dark Mode.chromium.png|0| Added|
|Charts-LineChart.Multiple RTL.chromium.png|0| Added|
|Charts-LineChart.Multiple.chromium.png|0| Added|

</details>


  <details><summary><b>Charts-SparkLineChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Charts-SparkLineChart.Basic Dark Mode.chromium.png|0| Added|
|Charts-SparkLineChart.Basic RTL.chromium.png|0| Added|
|Charts-SparkLineChart.Basic.chromium.png|0| Added|

</details>


  <details><summary><b>Charts-VerticalBarChart</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Charts-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|0| Added|
|Charts-VerticalBarChart.Basic Secondary Y Axis Dark Mode.chromium.png|0| Added|
|Charts-VerticalBarChart.Basic Secondary Y Axis RTL.chromium.png|0| Added|
|Charts-VerticalBarChart.Date Axis Vbc Dark Mode.chromium.png|0| Added|
|Charts-VerticalBarChart.Date Axis Vbc RTL.chromium.png|0| Added|
|Charts-VerticalBarChart.Date Axis- VBC.chromium.png|0| Added|
|Charts-VerticalBarChart.Dynamic - Wrap Labels.chromium.png|0| Added|
|Charts-VerticalBarChart.Dynamic Wrap Labels Dark Mode.chromium.png|0| Added|
|Charts-VerticalBarChart.Dynamic Wrap Labels RTL.chromium.png|0| Added|
|Charts-VerticalBarChart.Rotated Label Hide Legends Dark Mode.chromium.png|0| Added|
|Charts-VerticalBarChart.Rotated Label Hide Legends RTL.chromium.png|0| Added|
|Charts-VerticalBarChart.Rotated Label- Hide Legends.chromium.png|0| Added|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay RTL.chromium.png|1171| Changed|
|Drawer.overlay drawer full.chromium.png|1150| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>